src/libs/ReportUtils.ts

```diff
@@ -570,9 +570,6 @@ function getRootParentReport(report: OnyxEntry<Report> | undefined | EmptyObject
     return getRootParentReport(!isEmptyObject(parentReport) ? parentReport : null);
 }
 
-/**
- * @deprecated Use withOnyx or Onyx.connect() instead
- */
 function getPolicy(policyID: string | undefined): Policy | EmptyObject {
```

Comment:
Add a comment to mark that this function is only used in this file

Reply:
@DylanDylann I checked check other places, we only need to comment on what the function does.
ex: getParentReport, isOneTransactionReport